src/Umbraco.Web.UI/Views/Partials/Grid/Bootstrap2-Fluid.cshtml

```diff
@@ -64,21 +64,32 @@
         JObject cfg = contentItem.config;
 
         if(cfg != null)
-            foreach (JProperty property in cfg.Properties()) {
-                attrs.Add(property.Name + "='" + property.Value.ToString() + "'");
+            foreach (JProperty property in cfg.Properties())
+            {
+                var propertyValue = TemplateUtilities.CleanForXss(property.Value.ToString());
+                if (string.IsNullOrWhiteSpace(propertyValue) == false)
```

Comment:
I don't understand the changes everywhere to not add the property name if the value is an empty string, we didn't use to do this before and I am afraid that properties will now go missing when people were counting on them being there before.